src/ralph/lib/mixins/fields.py

```diff
@@ -61,6 +61,63 @@ class NullableCharField(
     pass
 
 
+# NOTE(romcheg): NUMP stands for No Useles Migrations Please.
+#                It is recommended to refrain from checking the other meaning
+#                in UD.
+class NUMPFieldMixIn(object):
+
+    def __init__(self, fields_to_ignore, *args, **kwargs):
+        super(NUMPFieldMixIn, self).__init__(*args, **kwargs)
+        self.fields_to_ignore = fields_to_ignore
+
+    def deconstruct(self):
+        name, path, args, kwargs = super(NUMPFieldMixIn, self).deconstruct()
+
+        if not self.__class__.__mro__.index(NUMPFieldMixIn) > 1:
+            # NOTE(romcheg): Exclude all fields that should not be concidered
+            #                when generating migrations.
+            kwargs = {
+                f: kwargs[f] for f in kwargs if f not in self.fields_to_ignore
+            }
+            path = '{}.{}'.format(NUMP.__module__, NUMP.__name__)
+
+            args = [self.base_class(*args, **kwargs)]
```

Comment:
Maybe we could use names like `nump_args`, `nump_kwargs` here to additionally point that those are args/kwargs for NUMP?

Reply:
Done